salt/haproxy/config/haproxy.cfg.jinja

```diff
@@ -117,6 +117,15 @@ frontend main
     bind :::80
     bind 127.0.0.1:19001  # This is our TLS socket.
 
+    # Client rate limiting
+    # See: https://www.haproxy.com/blog/four-examples-of-haproxy-rate-limiting
+    stick-table type ip size 100k expire 15s store http_req_rate(10s)
+    http-request track-sc0 src
+    http-request deny deny_status 429 if { sc_http_req_rate(0) gt 20 }
+
+    # Do we need to whitelist the LBs?
+    {# acl whitelist src 10.132.111.89 10.132.109.52 #}
```

Comment:
Shouldn't need to since haproxy is running on the load balancers... right?

Reply:
[86785be](https://github.com/python/psf-salt/pull/509/commits/86785be6deedb11aeab9d6d46dfdb044beb31c4c)